package.json

```diff
@@ -42,5 +43,11 @@
     "mocha": "^5.2.0",
     "promise-log": "^0.1.0",
     "validator": "^5.2.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint",
+      "pre-push": "npm run test"
```

Comment:
this may be a bit too much to demand, unfortunately the scraper will always break from time to time, and this would prevent pushing legitimate fixes when unrelated tests fail

Reply:
You can [bypass](https://github.com/typicode/husky/issues/124#issuecomment-299847973) husky(git hooks) by adding --no-verify to the git command.
i.e: `git commit --no-verify` and git `push --no-verify`